components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.ts

```diff
@@ -38,34 +43,40 @@ export class ServicesSidebarComponent implements OnInit {
     this.serviceGroupName$ = this.store.select(createSelector(serviceGroupState,
       (state) => state.selectedServiceGroupName));
 
-    this.servicesHealthSummary$ = this.store.select(createSelector(serviceGroupState,
+    this.svcHealthSummary$ = this.store.select(createSelector(serviceGroupState,
       (state) => state.servicesHealthSummary));
+    this.svcHealthSummary$.subscribe((servicesHealthSummary) => {
+      this.servicesHealthSummary = servicesHealthSummary;
+      this.totalServices = getOr(0, this.selectedHealth, this.servicesHealthSummary);
+    });
 
-    this.currentPage = 1;
```

Comment:
This page reset to default is now happening inside the `openServicesSidebar()` functions on the service-groups.component.ts. (https://github.com/chef/automate/pull/218/files#diff-f10f08039008f57ac8b564e592765087R154)

Why the change? - Because the OnInit function happens only once when the component is created in the browser, and not every time the bar is visible or invisible.